src/StochDynamicProgramming.jl

```diff
@@ -12,12 +12,13 @@ module StochDynamicProgramming
 
 using JuMP, Distributions
 
-export solve_SDDP, NoiseLaw, simulate_scenarios,
+export solve_SDDP, 
+            NoiseLaw, simulate_scenarios,
         SDDPparameters, LinearDynamicLinearCostSPmodel, set_state_bounds,
-        PiecewiseLinearCostSPmodel,
+        PiecewiseLinearCostSPmodel, extensive_formulation,
         PolyhedralFunction, NextStep, forward_simulations,
         StochDynProgModel, SDPparameters, sdp_optimize,
-        sdp_forward_simulation, sampling
+        sdp_forward_simulation, sampling 
```

Comment:
Aie ! 

You add blank space at the end of some lines. That's because of your text editor. I think you could parametrize it to avoid this kind of problem.